utilities/__init__.py

```diff
@@ -0,0 +1,2 @@
+from utilities.callbacks import default_callbacks
```

Comment:
do we expect this package/repo to be installed alongside other env, as this very generic name may easily collide...